src/status_im/common/signals/events.cljs

```diff
@@ -84,7 +84,9 @@
       (local-notifications/process cofx (transforms/js->clj event-js))
 
       "community.found"
-      (link-preview/cache-community-preview-data (transforms/js->clj event-js))
+      (let [community (transforms/js->clj event-js)]
+        (link-preview/cache-community-preview-data community)
+        (rf/dispatch [:discover-community/maybe-found-unknown-contract-community community]))
```

Comment:
if we still want to use dispatch it should be :fx :dispatch

Reply:
I've updated the code, do you mean like this?